src/Logging/Logging.Console/src/ConsoleLoggerOptions.cs

```diff
@@ -37,5 +37,10 @@ public ConsoleLoggerFormat Format
         /// Gets or sets format string used to format timestamp in logging messages. Defaults to <c>null</c>.
         /// </summary>
         public string TimestampFormat { get; set; }
+
+        /// <summary>
+        /// Gets or sets indication whether or not UTC timezone should be used to for timestamps in logging messages. Defaults to <c>false</c>.
+        /// </summary>
+        public bool UseUtcTimezone { get; set; } = false;
```

Comment:
```suggestion
        public bool UseUtcTimezone { get; set; }
```
`false` is already the default.

Reply:
True but I wanted to be more explicit - However it doesn't really make sense so I'll remove it!